projects/rocprofiler-systems/source/lib/core/trace_cache/storage_parser.hpp

```diff
@@ -152,7 +152,6 @@ class storage_parser
         ifs.close();
         ROCPROFSYS_DEBUG("File parsing finished. Removing %s from file system.\n",
                          m_filename.c_str());
-        std::remove(m_filename.c_str());
     }
 
```

Comment:
```suggestion
ifs.close();
}

```
We don't need print message anymore.